docs/how_to/extending_page_title.rst

```diff
@@ -291,14 +293,7 @@ low-level API to edit the toolbar according to your needs::
                 # Nothing to do
                 return
 
-            # check global permissions if CMS_PERMISSION is active
-            if get_cms_setting('PERMISSION'):
-                has_global_current_page_change_permission = has_page_change_permission(self.request)
-            else:
-                has_global_current_page_change_permission = False
-                # check if user has page edit permission
-            can_change = self.request.current_page and self.request.current_page.has_change_permission(self.request)
-            if has_global_current_page_change_permission or can_change:
+            if user_can_change_page(user=self.request.user, page=self.page):
```

Comment:
Does it make a difference whether this uses `self.page` or `self.request.current_page`?